app/db/stores/WorkoutStore.ts

```diff
@@ -34,6 +31,11 @@ export const WorkoutStoreModel = types
     get rootStore(): RootStore {
       return getParent(store) as RootStore
     },
+    get sortedWorkouts(): Workout[] {
```

Comment:
Are we currently ordering workouts by order of insertion?
If so, we should probably order by date on addition and avoid this here